### PR TITLE
Layout Editor: Avoid too many viewer refreshes

### PR DESCRIPTION
**Pull request by maurofmferrao**
relates to xibosignageltd/xibo-private#547

### CODE REVIEW

File: ui/src/editor-core/widget.js [hunk 1]
```diff
@@ -762,13 +762,33 @@ Widget.prototype.saveElements = function(
...
+
+          // Remove canvas from viewer
+          app.viewer.DOMObject.find('.designer-region-canvas').remove();
         });
     } else if (removeCurrentWidget) {
       // Remove widget
       app.layout.deleteObject('widget', this.widgetId)
         .then(() => {
-          reloadLayout(true);
+          debugger;
```
Comment (dasgarner) [inline ui/src/editor-core/widget.js:782]: I don't think you want this in here...

**Review by dasgarner — CHANGES_REQUESTED** [review]
Making sure I am paying attention eh? 😄

**Review by dasgarner — APPROVED** [review]

**Comment by maurofmferrao** [comment]
TEST: *

**Comment by maurofmferrao** [comment]
TEST: * END

**Comment by maurofmferrao** [comment]
TEST: layout_designer_empty.cy.js, layout_designer_populated_unchanged.cy.js, layout_designer_populated.cy.js END

**Pull request merged**